Makefile

```diff
@@ -180,34 +170,26 @@ clean:
 ############################################
 images: registry-image k8s-broker-image user-broker-image controller-image
 
-registry-image: contrib/registry/Dockerfile
-	SC_GOOS=linux SC_GOARCH=amd64 BINDIR=bin/linux_amd64 $(DOCKER_FLAG) \
-	  $(MAKE) bin/linux_amd64/registry
-	cp contrib/registry/Dockerfile $(BINDIR)/linux_amd64/
-	cp contrib/registry/data/charts/*.json $(BINDIR)/linux_amd64/
-	docker build -t registry:$(VERSION) $(BINDIR)/linux_amd64
-	rm -f $(BINDIR)/linux_amd64/*.json
-
-k8s-broker-image: contrib/broker/k8s/Dockerfile
-	SC_GOOS=linux SC_GOARCH=amd64 BINDIR=bin/linux_amd64 $(DOCKER_FLAG) \
-	  $(MAKE) bin/linux_amd64/k8s-broker
-	cp contrib/broker/k8s/Dockerfile $(BINDIR)/linux_amd64/
-	docker build -t k8s-broker:$(VERSION) $(BINDIR)/linux_amd64
-	rm -f $(BINDIR)/linux_amd64/Dockerfile
-
-user-broker-image: contrib/broker/user_provided/Dockerfile
-	SC_GOOS=linux SC_GOARCH=amd64 BINDIR=bin/linux_amd64 $(DOCKER_FLAG) \
-	  $(MAKE) bin/linux_amd64/user-broker
-	cp contrib/broker/user_provided/Dockerfile $(BINDIR)/linux_amd64/
-	docker build -t user-broker:$(VERSION) $(BINDIR)/linux_amd64
-	rm -f $(BINDIR)/linux_amd64/Dockerfile
-
-controller-image: pkg/controller/catalog/Dockerfile
-	SC_GOOS=linux SC_GOARCH=amd64 BINDIR=bin/linux_amd64 $(DOCKER_FLAG) \
-	  $(MAKE) bin/linux_amd64/controller
-	cp pkg/controller/catalog/Dockerfile $(BINDIR)/linux_amd64/
-	docker build -t controller:$(VERSION) $(BINDIR)/linux_amd64
-	rm -f $(BINDIR)/linux_amd64/Dockerfile
+registry-image: contrib/registry/Dockerfile $(BINDIR)/registry
+	cp contrib/registry/Dockerfile $(BINDIR)
+	cp contrib/registry/data/charts/*.json $(BINDIR)
+	docker build -t registry:$(VERSION) $(BINDIR)
+	rm -f $(BINDIR)/*.json
```

Comment:
That was missing prior to this change, but I don't mind fixing that here if you don't mind me doing so.